tests/scripts/kubeadm-dind.sh

```diff
@@ -7,23 +7,22 @@ tarname=image.tar
 tarfile=${WORK_DIR}/tests/${tarname}
 
 copy_image_to_cluster() {
-    local build_image=$1
-    local final_image=$2
+    local final_image=$1
 
     mkdir -p ${WORK_DIR}/tests
-    docker save -o ${tarfile} ${build_image}
+    docker save -o ${tarfile} ${final_image}
     for c in kube-master kube-node-1 kube-node-2; do
         docker cp ${tarfile} ${c}:/
         docker exec ${c} /bin/bash -c "docker load -i /${tarname}"
-        docker exec ${c} /bin/bash -c "docker tag ${build_image} ${final_image}"
     done
 }
 
 copy_rbd() {
     for c in kube-master kube-node-1 kube-node-2; do
         docker cp ${scriptdir}/dind-cluster-rbd ${c}:/bin/rbd
         docker exec ${c} /bin/bash -c "chmod +x /bin/rbd"
-        docker exec ${c} /bin/bash -c "docker pull ceph/base"
+        # hack for Azure, after vm is started first docker pull command fails intermittently
+        docker exec ${c} /bin/bash -c "docker pull ceph/base || true"
```

Comment:
the script is running with `-e` mode. if docker exec fails will the whole script to fail? do you need to wrap it with `set +e`